next.config.js

```diff
@@ -32,6 +32,15 @@ const config = {
   images: {
     domains: ["images.ctfassets.net"],
   },
+  async redirects() {
+    return [
+      {
+        source: "/:address",
+        destination: "/ethereum/:address",
```

Comment:
we can double check with the rest of the team to find out their thoughts on this because there are multiple ways to implement it. Also query params are an options since it doesn't break the routing logic we already use